README.md

```diff
@@ -943,6 +943,11 @@ expect('div').toHaveDescendant('.child');
 expect('div').toHaveDescendantWithText({selector: '.child', text: 'text'});
 ```
 
+Please notice that due to restrictions within Jest (not applying actual layout logic), certain matchers may give false positives - eg.:
```

Comment:
Can we make it short and clear, please? I think one sentence is enough.

Reply:
Agreed... not sure if my rewording is short (enough)?